tifeatures/dependencies.py

```diff
@@ -130,17 +226,18 @@ def properties_query(
 
 def filter_query(
     query: Optional[str] = Query(None, description="CQL2 Filter", alias="filter"),
-    filter_lang: Optional[FilterLang] = Query(
-        FilterLang.cql2_text,
-        description="CQL2 Language (cql2-text, cql2-json)",
+    filter_lang: Optional[enums.FilterLang] = Query(
+        None,
```

Comment:
we don't have to set a default here because we always default to cql2-text in the code later